com.unity.render-pipelines.high-definition/Editor/Wizard/HDWizard.Window.cs

```diff
@@ -246,22 +253,41 @@ static HDWizard()
         static void WizardBehaviourDelayed()
         {
             if (frameToWait > 0)
-                --frameToWait;
-            else
             {
-                EditorApplication.update -= WizardBehaviourDelayed;
+                --frameToWait;
+                return;
+            }
 
-                if (HDProjectSettings.wizardIsStartPopup && !HDUserSettings.wizardPopupAlreadyShownOnce)
-                {
-                    //Application.isPlaying cannot be called in constructor. Do it here
-                    if (Application.isPlaying)
-                        return;
+            // No need to update this method, unsubscribe from the application update
+            EditorApplication.update -= WizardBehaviourDelayed;
 
-                    OpenWindow();
-                }
+            //Application.isPlaying cannot be called in constructor. Do it here
+            if (Application.isPlaying)
+                return;
 
-                EditorApplication.quitting += () => HDUserSettings.wizardPopupAlreadyShownOnce = false;
+            EditorApplication.quitting += () => HDUserSettings.wizardPopupAlreadyShownOnce = false;
+
+            ShowWizardFirstTime();
+        }
+
+        static void ShowWizardFirstTime()
+        {
+            // Unsubscribe from possible events
+            // If the event has not been registered the unsubscribe will do nothing
+            RenderPipelineManager.activeRenderPipelineTypeChanged -= ShowWizardFirstTime;
+            
+            if (!CanShowWizard())
+            {
+                // Delay the show of the wizard for the first time that the user is using HDRP
+                RenderPipelineManager.activeRenderPipelineTypeChanged += ShowWizardFirstTime;
+                return;
             }
+
+            // If we reach this point can be because
+            // - That the user started Unity with HDRP in use
+            // - That the SRP has changed to HDRP for the first time in the session
+            if (!HDUserSettings.wizardPopupAlreadyShownOnce) 
```

Comment:
HDProjectSettings.wizardIsStartPopup is not checked anymore.
Have you tested if this bevahe correctly when you check/uncheck the checkbox at the end of the window?

Reply:
Check my other comment